File: src/main/java/pe/nanamochi/banchus/repositories/BeatmapsetRepository.java
package pe.nanamochi.banchus.repositories;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import pe.nanamochi.banchus.entities.db.Beatmapset;

@Repository
public interface BeatmapsetRepository extends JpaRepository<Beatmapset, Integer> {
  Beatmapset findById(int id);
}

File: banchus-app/src/main/kotlin/pe/nanamochi/banchus/beatmap/repository/BeatmapsetRepository.kt
package pe.nanamochi.banchus.beatmap.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import pe.nanamochi.banchus.beatmap.entity.Beatmapset

@Repository
interface BeatmapsetRepository : JpaRepository<Beatmapset, Int> {
    fun findBeatmapsetById(id: Int): Beatmapset?
}
